crates/matrix-sdk-ui/src/room_list_service/filters/fuzzy_match_room_name.rs

```diff
@@ -44,7 +44,7 @@ impl FuzzyMatcher {
 ///
 /// Rooms are fetched from the `Client`. The pattern and the room names are
 /// normalized with `normalize_string`.
-pub fn new_filter(pattern: &str) -> impl Filter {
+pub fn new_filter(pattern: &str) -> impl Filter + use<> {
```

Comment:
TIL https://blog.rust-lang.org/2024/09/05/impl-trait-capture-rules/

Reply:
I find it particularly frustrating to add empty `use`… I don't understand why it's not the default.